lib/ProductOpener/KnowledgePanels.pm

```diff
@@ -1290,6 +1290,26 @@ sub create_ingredients_analysis_panel ($product_ref, $target_lc, $target_cc, $op
 	return;
 }
 
+sub remove_latex_sequences ($string) {
+
+	# Some wikipedia abstracts have chemical formulas like {\\displaystyle \\mathrm {NaNO_{3}} +\\mathrm {Pb} \\to \\mathrm {NaNO_{2}} +\\mathrm {PbO} }
+	# In practice, we remove everything between { }
+
+	$string =~ s/
+        (                   
+        {                   # match an opening {
+            (?:
+                [^{}]++     # one or more non angle brackets, non backtracking
+                  |
+                (?1)        # found { or }, so recurse to capture group 1
+            )*
+        }                   # match a closing }
+        )                   
+        //xg;
```

Comment:
So good you used extended format for regexp :heart:

Reply:
@alexgarel Well I adapted some code I found on Stack Overflow that had this extended format which is indeed quite cool. That's after ChatGPT gave me 3 wrong answers. ;)